c_emulator/riscv_platform_impl.h

```diff
@@ -17,6 +17,11 @@ extern bool rv_enable_dirty_update;
 extern bool rv_enable_misaligned;
 extern bool rv_mtval_has_illegal_inst_bits;
 
+extern int  rv_enable_Smepmp;
+extern int  rv_enable_Zicond;
```

Comment:
Why are these int? And why are you using uppercase names?

Reply:
When using getopt,  the target variable to be set is defined as an int.
Therefore,  to avoid a warning,  they are ints.  They are converted to bool
before being returned to the Sail calling functions.

Reply:
Right... :(